### PR TITLE
Add delete free time command

### PR DESCRIPTION
**Pull request by larainezo**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteTimeCommand.java [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+import seedu.address.model.tag.FreeTimeTag;
+
+/**
+ * Deletes a free time to an existing person in the address book.
+ */
+public class DeleteTimeCommand extends Command {
+
+    public static final String COMMAND_WORD = "deleteTime";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Deletes a free time to the person identified "
```
Comment (donwong2308) [inline src/main/java/seedu/address/logic/commands/DeleteTimeCommand.java:33]: So the command is
Eg: `deleteTime 1 ft/Mon:1300-1400` 
Above is correct?

**Review by donwong2308 — APPROVED** [review]
LGTM for the most part.
Will be good to create issue for the test case for the `deleteTime` 
and modify the `createEditedPerson` method in AddTimeCommand class to be shorter.

**Review by JustWeiHao — APPROVED** [review]
LGTM

**Review by nyankoclaws — APPROVED** [review]
LGTM

**Pull request merged**